plugin/storage/es/spanstore/reader.go

```diff
@@ -83,20 +85,33 @@ var (
 	tagFieldList = []string{tagsField, processTagsField, logFieldsField}
 )
 
+type spanReaderMetrics struct {
```

Comment:
these looks like generic metrics for a SpanReader - don't we already have a metrics wrapper for storage implementations?